lib/CMakeLists.txt

```diff
@@ -55,42 +70,47 @@ add_library(clspv_core
   ${CMAKE_CURRENT_SOURCE_DIR}/ZeroInitializeAllocasPass.cpp
 )
 
-# Enable C++11 for our executable
-target_compile_features(clspv_core PRIVATE cxx_range_for)
+foreach(clspv_lib clspv_core clspv_passes)
+  # Enable C++11 for our executable
+  target_compile_features(${clspv_lib} PRIVATE cxx_range_for)
 
-target_include_directories(clspv_core PRIVATE ${SPIRV_HEADERS_INCLUDE_DIRS})
+  target_include_directories(${clspv_lib} PRIVATE ${SPIRV_HEADERS_INCLUDE_DIRS})
 
-target_include_directories(clspv_core PRIVATE ${LLVM_INCLUDE_DIRS})
+  target_include_directories(${clspv_lib} PRIVATE ${LLVM_INCLUDE_DIRS})
 
-target_include_directories(clspv_core PRIVATE ${CLANG_INCLUDE_DIRS})
+  target_include_directories(${clspv_lib} PRIVATE ${CLANG_INCLUDE_DIRS})
 
-target_include_directories(clspv_core PUBLIC ${CLSPV_INCLUDE_DIRS})
+  target_include_directories(${clspv_lib} PUBLIC ${CLSPV_INCLUDE_DIRS})
+
+  add_dependencies(${clspv_lib} clspv_baked_opencl_header clspv_c_strings clspv_glsl)
```

Comment:
I'm not certain these dependencies are required for clspv_passes.

Reply:
Adjusted them.  One is needed by `clspv_core`.  The other two are needed by `clspv_passes`.  Added comments to clarify.